README.md

```diff
@@ -80,6 +80,14 @@ Use `basin publication start` to start a daemon that will continuously push chan
 basin publication start --private-key [PRIVATE_KEY] namespace.relation_name
 ```
 
+## Upload a Parquet file
+
+Use `basin publication upload` to upload a Parquet file.
```

Comment:
would be good to include some context about first needing to create the publication. which, right now, will be a bit weird because the user will need a connected postrgres for that. for a user that just wants to push parquet files, how should they go about creating a publication? maybe with `publication create`, if you don't specify a `--dburi` then it will just create it remotely? in which case, if the user does give a `--dburi`, the CLI should check if it already exists remotely somehow so that it doesn't create the local postgres publication and then error when creating the remote publication.

Reply:
wait, there's a problem. looks like we need to get the schema from the parquet file as well

Reply:
feels off. why would we pass the schema, if we're not even writing to the database?

Reply:
just don't send the schema? this is a bit of a moving target... we can rethink this for a better design, just thinking how we can make it "work" for now